bridge_adaptivity/api/backends/api_client.py

```diff
@@ -7,19 +7,19 @@
 from api.backends.base_api_client import BaseApiClient
 from api.backends.dart_api_client import DartApiClient
 from api.backends.edx_api_client import OpenEdxApiClient
-from bridge_lti.models import LtiConsumer
+from bridge_lti.models import LtiContentSources
```

Comment:
Plural in model's name looks odd. Please improve.

Reply:
I fixed it.